src/Lightning.ts

```diff
@@ -324,7 +320,7 @@ export const LightningMixin = (superclass) => class extends superclass {
         }
 
         if (String(payeeUser._id) === String(this.user._id)) {
-          const error = 'User tried to pay himself'
+          const error = 'User tried to pay themselves'
```

Comment:
User is singular here. are you sure it should be themselves?

Reply:
Themself can be used as well (it's usage is more recent), however for formal purposes, themselves is generally preferred. See [this](https://www.lexico.com/grammar/themselves-or-themself).